src/ebay-menu-button/README.md

```diff
@@ -0,0 +1,57 @@
+# ebay-menu-button
+
+## Demo
+[Storybook](https://pages.github.corp.ebay.com/React/ebayui-core-react/master/?path=/story/ebay-menu-button--default)
+
+## Install
+```
+yarn add ebayui-core-react
+```
+
+## Usage
+```jsx harmony
+import {
+    EbayMenuButton,
+    EbayMenuButtonItem as Item,
+    EbayMenuButtonSeparator as Separator,
+    EbayMenuButtonLabel as Label
+} from 'ebayui-core-react/ebay-menu-button';
+import '@ebay/skin/menu-button';
+<EbayMenuButton text="Menu">
+    <Item>Item 1</Item>
+    <Item>Item 2</Item>
+    <Item>Item 3</Item>
+    <Separator />
+    <Item>Log in</Item>
+</EbayMenuButton>
+// With a complext label
+<EbayMenuButton a11yText="Menu">
+    <Label><img src="menu-logo.svg" /></Label>
```

Comment:
With custom label maybe?

Even it's for README, I would still suggest to use accessible example:

```tsx
<EbayMenuButton a11yText="Menu">
    <Label><img src="menu-logo.svg" alt="Plus program logo" /></Label>
```

or keep it simple like [Marko ebay-menu-button example](https://github.com/eBay/ebayui-core/tree/v10.1.0/src/components/ebay-menu-button#label-usage):

```tsx
<EbayMenuButton a11yText="Menu">
    <Label><div>Custom Label</div></Label>
```